### PR TITLE
Frontend: Login form

### LINKED ISSUE #49: Frontend: Login form
**__Task Description__**
Describe the task to be completed.

**__Technical Considerations__**
Include any technical considerations including architecture (e.g. API), required libraries, etc.

**__Additional Considerations__**
Any supplemental information including unresolved questions, links to external resources, screenshots, etc.

### PR DESCRIPTION
**Pull request by 12cassie34**
For the issue https://github.com/chingu-voyages/v40-bears-team-37/issues/49 .

The RWD and the carousel will be in other issues.

### CODE REVIEW

File: client/src/routes/Login.tsx [hunk 1]
```diff
@@ -1,32 +1,116 @@
...
+                    )}
+                  </InputFormStyles>
+                );
+              }) }
+              <button type="submit">Login</button>
+            </form>
+          )
+          : (
+            <>
+              <p>Thank you for login.</p>
```
Comment (cherylli) [inline client/src/routes/Login.tsx]: this part seems to be is unreachable, on sucessful login, user will just be redirected to the calendar

**Review by cherylli — CHANGES_REQUESTED** [review]
Also, when I use a random email/password combination (not a registered user), it logs me in.

File: client/src/routes/Login.tsx (hunk 1 above)
Comment (12cassie34) [reply client/src/routes/Login.tsx]: I'm going to delete it.

**Review by amirfakhrullah — DISMISSED** [review]

**Review by cherylli — APPROVED** [review]
works fine and error is showing now, but all axios errors on login will just be "Email or password is invalid."?

**Pull request merged**